src/controllers/loan-manager/redux/actions.ts

```diff
@@ -106,6 +108,7 @@ export function loadLoanAccounts(account: EdgeAccount): ThunkAction<Promise<void
       // Instantiate loan account from loanAccountEntry
       const loanAccount = await makeLoanAccount(borrowPlugin, wallet)
       loanAccount.closed = loanAccountEntry.closed
+      loanAccount.shown = loanAccountEntry.shown
```

Comment:
Make sure to not startEngine if `shown` is false. Also rename to `active` because of this logic.

Reply:
We went with a simpler approach which didn't require a new field.